docs/fluent.md

```diff
@@ -0,0 +1,70 @@
+# Fluent API
+
+The Fluent API is the most basic component of the BDK. This component provides the developers very quickly an entry point 
+to discover all the features of the BDK, helps them to easily understand how to make a bot interacting with the 
+Symphony platform.
+
+## SymphonyBdk
+
+The heart of the Fluent API is the [`SymphonyBdk`](../symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java).
+This component is an entry point for a developer to go through all the features of the BDK. A `SymphonyBdk` object is 
+built from the information extracting from the BDK configuration file.
+
+```java
+public class Example {
+    
+    public static void main(String[] args) {
+        //Init the BDK entry point
+        final SymphonyBdk bdk = new SymphonyBdk(loadFromSymphonyDir("config.yaml"));
```

Comment:
I'd prefer to have`loadFromClasspath("/config.yaml")` in doc examples

Reply:
ah ha: 😄 
https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blame/master/docs/activity.md#L17

Do you want me to update this one as well?